src/rgw/rgw_putobj_processor.cc

```diff
@@ -180,7 +180,7 @@ int ManifestObjectProcessor::next(uint64_t offset, uint64_t *pstripe_size)
   if (r < 0) {
     return r;
   }
-  r = writer.set_stripe_obj(stripe_obj);
+  r = writer.set_stripe_obj(std::move(stripe_obj));
```

Comment:
std::move() here is only casting the type of stripe_obj to `rgw_raw_obj&&`. but passing that to `set_stripe_obj(const rgw_raw_obj& raw_obj)` casts it back and results in a copy instead of move. so i don't think this change has any effect

Reply:
Thank you, I will try #30746 pr

Reply:
#30746 PR fixes this problem. close this PR now. thanks!